apps/single-view/src/Views/SearchView/searchResults.tsx

```diff
@@ -49,54 +54,117 @@ export const SearchResults = (props: myProps): JSX.Element => {
     }
   };
 
+  const selectMatch = (person: housingSearchPerson) => {
+    if (!person.isSelected) {
+      person.isSelected = true;
+      setSelectedRecords([...selectedRecords, person]);
+    } else {
+      setSelectedRecords(
+        selectedRecords.filter((p) => {
+          return p != person;
+        })
+      );
+      person.isSelected = false;
+    }
+  };
+
+  const mergeSelectedRecords = async (records: housingSearchPerson[]) => {
+    try {
+      setMergeError(false);
+      const sv_id = await mergeRecords(records);
+      return (window.location.href = `/customers/single-view/${sv_id}`);
```

Comment:
is this an interim solution?

I feel like we should just be going to `customers/${sv_id}`